tests/FSharp.Compiler.ComponentTests/TypeChecks/Graph/DependencyResolutionTests.fs

```diff
@@ -1,11 +1,14 @@
 ﻿module TypeChecks.DependencyResolutionTests
 
 open TypeChecks.TestUtils
-open NUnit.Framework
+open Xunit
 open FSharp.Compiler.GraphChecking
 open Scenarios
 
-[<TestCaseSource(nameof scenarios)>]
+let scenarios = scenarios |> Seq.map (fun p -> [| box p |])
+
+[<Theory>]
```

Comment:
Note that attribute was forgotten here so tests were not executing - they are now.